.circleci/config.yml

```diff
@@ -15,17 +15,22 @@ jobs:
 
       # Download and cache dependencies
       - restore_cache:
-          keys:
-          - v1-dependencies-{{ checksum "package.json" }}
-          # fallback to using the latest cache if no exact match is found
-          - v1-dependencies-
+          key: v2-dependencies-{{ checksum "yarn.lock" }}
```

Comment:
is the v2 to split off potential cache hash collisions?

Reply:
Yep, it's a cache buster. Likely completely redundant to bust here as collisions should be nearly impossible, but it's cheap to bust the cache.